enarx-mirrorlabels

```diff
@@ -0,0 +1,173 @@
+#!/usr/bin/python3
+# SPDX-License-Identifier: Apache-2.0
+
+import enarxbot
+
+# Dictionary of labels to be mirrored across all repos.
+ORGANIZATION_LABELS = {
+    'amd sev': {
+        'description': 'Issues related to AMD SEV',
+        'color': '000000',
+        'old_names': {},
+    },
```

Comment:
```suggestion
    'amd sev': {
        'description': 'Issues related to AMD SEV',
        'color': '000000',
    },
```

`old_names` should be optional. Also, why is it a `dict`?

Reply:
1) I agree.
2) Using a list or tuple made readability a lot worse.